x-pack/plugins/security_solution/server/fleet_integration/handlers/validate_endpoint_package_policy.ts

```diff
@@ -9,26 +9,29 @@ import moment from 'moment';
 
 import type { NewPackagePolicyInput } from '@kbn/fleet-plugin/common';
 
-export const validateEndpointPackagePolicy = (input: NewPackagePolicyInput) => {
-  if (input.config?.policy?.value?.global_manifest_version) {
+export const validateEndpointPackagePolicy = (inputs: NewPackagePolicyInput[]) => {
+  const input = inputs.find((i) => i.type === 'endpoint');
+  if (input?.config?.policy?.value?.global_manifest_version) {
```

Comment:
is there an `else` case missing here, if there is no `endpoint` input at this point? Not sure if that's a possible scenario @juliaElastic

Reply:
if endpoint input is missing, that would mean the package policy is in a broken state, shouldn't happen in normal circumstances

Reply:
Thanks @juliaElastic And is there any case where inputs array can contain another input from other type? Or there is only a single entry for endpoint in this array in a normal scenario?

Reply:
I'm not familiar with all the details of the endpoint package, only that when I install the latest version of package, it has only one input type:endpoint. Some packages define more than one inputs, e.g. system.